src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_wed calendar--month-length_29">
```

Comment:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/102411130/230848758-0507e0df-2c57-4c7e-bf37-a276bc04dfb7.png">
proofread your code with the checklist